### ISSUE TITLE
Paraboloid example gives RoundoffLimited error

### ISSUE THREAD

**Issue by my1e5** [relf/slsqp#4]
Running `cargo run --example paraboloid` gives 

```bash
    Finished dev [unoptimized + debuginfo] target(s) in 0.13s
     Running `target\debug\examples\paraboloid.exe`
Optim error: RoundoffLimited
```
Is this expected?

**Comment by relf** [comment]
Thanks for reporting. No it is not meant to show off like this! :smile: 
You can fix the example by changing the `maxeval` value (`100` instead of `200` should work properly).
The `RoundoffLimited` error means the optimizer cannot make any progress (you can also get the `x` and the `fmin` value when this error occurs (`Err((e, x_opt, y_opt))`).

**Comment by relf** [comment]
Fixed by 2567d5d0d354a60d93dd189fabbc28b243cf6e99
